src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -93,6 +93,25 @@ public function get_template_object() {
 			: tribe( Template\Event::class );
 	}
 
+	/**
+	 * Detemines wether we are in a Single event page or not,
+	 * base only on global context.
+	 *
+	 * @since  TBD
+	 *
+	 * @return bool
```

Comment:
Add return value description:
```suggestion
	 * @return bool Whether the current request is for the single event template or not.
```